Closure_Front_End/src/components/PieChart.vue

```diff
@@ -0,0 +1,22 @@
+
+<script>
+import { defineComponent } from 'vue';
+import { Pie } from 'vue3-chart-v2';
+
+export default defineComponent({
+  name: 'MonthlyChart',
+  extends: Pie,
+  mounted () {
+    // Overwriting base render method with actual data.
+    this.renderChart({
+      labels: ['Must', 'Choose from list', 'Choice'],
```

Comment:
Can we have `CornerStone` and `Complementary` as well?

Reply:
Sure, what are the numbers representing their types?

Reply:
i need this also for the progress bars, even though complementary courses would probably not appear there, as there is no quota of them that needs to be filled

Reply:
I'm not aware of numbers representing types on the backend, I guess someone created such a mapping somewhere in the FE

Reply:
The `Track` model does have the quota, is it missing from the API?